themes/default/templates/__overrides.html

```diff
@@ -16,11 +16,22 @@
 <!-- Page Title -->
 
 {% block head_page_title %}
-  {% if page_title %}{{page_title}} {% endif %}{% if site_title %}{{site_title}} « {% endif %}
+  {% if page_title %}{{page_title}} «  {% endif %}{% if site_title %}{{site_title}} {% endif %}
 {% endblock head_page_title %}
 
 <!-- Other Options -->
 
+{% block footer %}
```

Comment:
@jduss4 It's basically only there to demonstrate that you can override the footer, since I think that will be a very common activity.